docs/reference/cluster/voting-exclusions.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[voting-config-exclusions]]
+== Voting configuration exclusions API
+++++
+<titleabbrev>Voting Configuration Exclusions</titleabbrev>
+++++
+
+Adds or removes master-eligible nodes from the
+<<modules-discovery-voting,voting configuration exclusion list>>.
+
+[float]
+=== Request
+
+[source,js]
+--------------------------------------------------
+# Add a node to the voting configuration exclusions list 
+POST /_cluster/voting_config_exclusions/<node_name>
+
+# Remove all exclusions from the list
+DELETE /_cluster/voting_config_exclusions
+--------------------------------------------------
+// CONSOLE
+
+[float]
+=== Path parameters
+
+`node_name`::
+  A <<cluster-nodes,node filter>> that identifies {es} nodes.
+
+[float]
+=== Description
+  
+If the <<modules-discovery-settings,`cluster.auto_shrink_voting_configuration`
```

Comment:
I'm concerned that mentioning this setting so prominently here will encourage too many users to fiddle with it. It should be left set to the default of `true` in the overwhelming majority of clusters. We need it to be configurable to support some very unusual resilience requirements, but it's only part of the story for those cases.

I think it'd be better to word most of this description as if the setting is set to the default of `true`.

Reply:
Thanks for the feedback!  I've updated the API description to downplay that setting.